spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
+PYLOADCORE="/usr/local/python/bin/python /usr/local/pyload/share/pyload/pyLoadCore.py --pidfile=${PIDFILE}"
+export PATH="/usr/local/pyload/bin:$PATH"
+
+case $1 in
+    start)
+        if [ -f /usr/syno/etc/ssl/ssl.crt/server.crt -a \
```

Comment:
That might be, but the symlinks would only work if you run the package as root, and that should change anyway. 
An alternative is to copy the certificate files to your /var directory, correct the permissions, and use those for SSL access.